config/util/helpers.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
```

Comment:
this can also replace internal/util/util.go.

It's StringToPtr is slightly different (doesn't have the `s == ""` check, but the only place it's used (in engine.go) it won't matter.

Reply:
Missed one in internal/config/translate.go